.github/CODEOWNERS

```diff
@@ -7,6 +7,10 @@
 docs/compilers @dotnet/roslyn-compiler
 docs/ide @dotnet/roslyn-ide
 
+# Notify docs team for new breaking change documentation
+docs/compilers/CSharp @dotnet/docs
+docs/compilers/Visual\ Basic @dotnet/docs
+
```

Comment:
@jcouv @cston It doesn't seem this will work because dotnet/docs doesn't have write access to the repository.

![image](https://user-images.githubusercontent.com/31348972/156428916-0227535e-25b1-4bc3-9b71-d811b1699c56.png)

Per [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)

> The people you choose as code owners must have read permissions for the repository. When the code owner is a team, that team must be visible and it must have write permissions, even if all the individual members of the team already have write permissions directly, through organization membership, or through another team membership.

Reply:
I'll ping github to understand why *write* access is needed.
Also running a test with this fake PR: https://github.com/dotnet/roslyn/pull/59888